docs/UserGuide.adoc

```diff
@@ -32,9 +33,11 @@ The front will have a question prompt and the back will have an answer.
 Physical cards are bulky and hard to search for. They are also very difficult to organise. Flashy is the result of our desire to eliminate these challenges that arise from the use of conventional flashcards. Flashy helps you organise your flashcards using its *tagging* system. By associating flashcards to different tags, Flashy is able to retrieve all flashcards of a given tag in a flash.
 
 Studies have shown that *spaced repetition* is far more effective than regular studying.
-And thus we gave Flashy an intelligent scheduler to help space out your study sessions and make them more effective.
+And we know studying is hard, thus we gave Flashy an intelligent scheduler to help space out your study sessions and make them more effective.
 // comment: do we want to add Italisization for Flashy?
 // comment: how do we reference special words like spaced repetition?
+[NOTE]
+`Spaced Repetition` is the idea that spreading out study sessions is much more effective than cramming it ove rone night.
```

Comment:
over one*

Reply:
also maybe add link to wikipedia or something

Reply:
kk